dataedit/models.py

```diff
@@ -62,6 +62,16 @@ class Meta:
         unique_together = (("schema", "name"),)
 
 
+class TableGraph(models.Model):
+    name = CharField(max_length=50, null=False)
+    table = CharField(max_length=1000, null=False)
+    schema = CharField(max_length=1000, null=False)
+    column_x = CharField(max_length=200, null=False)
+    column_y = CharField(max_length=200, null=False)
+    filters_x = CharField(max_length=1000, null=False)
+    filters_y = CharField(max_length=1000, null=False)
```

Comment:
Why are filters for x and y separated? What does it mean to have a filter on x but not on y? Why are they strings?

Reply:
I just added the filter as an exemple, I don't actually know what the filter field should contain